### ISSUE TITLE
No resize options and Toggle button not working

### ISSUE THREAD

**Issue by AReallyNervousNurse** [YePpHa/YouTubeCenter#1700]
The video looked weird and had part of the page on top of the video, like this: https://github.com/YePpHa/YouTubeCenter/issues/1697
I went to the toggle menu for YTC and tried resizing the video and I accidentally ended up deleting all the Player Size options (Cause' I'm an idiot and I kept pressing the Discard button when I didn't want to make a new custom video size)
![problem with ytc](https://cloud.githubusercontent.com/assets/12164923/7385450/5bf1f6e6-edfb-11e4-918f-f44ab06fab7b.png)
Stuff I disabled like Dash Playback are still in effect but now the toggle button doesn't work. I try to click it but the menu doesn't even come up. I've heard it might be because of a new Youtube update. Any help would be appreciated.

While I was trying to fix the problem I switched my version. Before I was using the 2.1.7 version (Firefox add-on) and now I'm using the Developer Build 501 (Greasemonkey script). However, it didn't fix the problem at all.
My current Firefox version is 37.0.1

**Comment by SINE** [comment]
Pretty sure its related to
#1695
#1696

**Comment by AReallyNervousNurse** [comment]
Ah I see. Is this why I can't access the toggle menu? (Sorry, I'm new here).

**Comment by AReallyNervousNurse** [comment]
Ah I see. Is this why I can't access the toggle menu, or is that entirely another issue? (Sorry, I'm new and I don't know how to script at all)

**Comment by SINE** [comment]
The toggle menu is completely empty for you?
Sorry, I was confused because the functionality also doesn't work (the menu shows for me).

I'm using Firefox 37.0.2, YouTubeCenter 501 and the menu shows.

Try the following: 
* YouTubeCenter Settings > General > Reset Settings
* CTRL + SHIFT + DEL > CACHE + COOKIES

Then reload the YouTube page and give some feedback if that helped.

Also make sure (if you have the AddOn Stylish installed), to deactivate all AddOns that manipulate the website, for example Stylish, FireBug or other Greasemonkey scripts / YouTube addons.

**Comment by AReallyNervousNurse** [comment]
Sorry, I'm kind of confused by your comment. What's happening is that I click on the button (or gear) "Toggle Youtube Center Settings Panel" and nothing happens. The drop-down menu doesn't appear at all. No matter how many times I've clicked the menu doesn't pop up. Is there another way to get to the YoutubeCenter Settings?

**Comment by SINE** [comment]
So the YouTubeCenter Settings also don't work, interesting.
Did you try uninstalling the script and re-installing it?

I don't know of a way to access the settings otherwise.

**Comment by AReallyNervousNurse** [comment]
Hey, thank you so much! I didn't think to uninstall and then re-install it again. What I did was remove the script (Developer Build 501), disabled Greasemonkey and then I closed the window. I opened up Firefox again, re-installed the script and enabled Greasemonkey. I was happy to see the opening message for YTC (The message you get the first time you have the add-on). The videos work fine (except for a big black rectangle on the left but I believe that will be fixed) and the resize options work now.

Thank you so much for all the help. I really appreciate it.
